ql/src/test/queries/clientpositive/dataconnector_mysql.q

```diff
@@ -14,6 +14,9 @@ SHOW DATABASES;
 USE db_mysql;
 SHOW TABLES;
 
+SHOW CREATE TABLE country;
+SELECT * FROM country;
```

Comment:
Can you add similar tests for the other supported connectors? (Oracle, MSSQL, etc.). I would like to ensure that we are not breaking something else while fixing this problem.

Reply:
> Can you add similar tests for the other supported connectors? (Oracle, MSSQL, etc.). I would like to ensure that we are not breaking something else while fixing this problem.

Adding qtest for Oracle or MSSQL needs docker env like MySQL qtest. This test may be outside the scope of this pr but make sense. 
But i think we should optimize this fix to only modify method of subclasses(MySQL, Oracle, MSSQL) instead of modifing parent class ` AbstractJDBCConnectorProvider.java` repeatedly. Because it is difficult to ensure that all jdbc connectors have consistent table structures.
 wdyt. thx. @zabetak

Reply:
Ideally the tests should be present in the same commit that fixed HIVE-25091. It is a bit unfortunate that HIVE-25091 didn't add any tests. 

I cannot tell for sure if the best way to handle this kind of problem is through sub-classing cause the specs/docs for the CONNECTOR are somewhat incomplete and tests are almost nonexistent.

I agree with you that tests are not necessarily in the scope of this PR. However, for moving forward with this or other fixes we should have a minimal set of tests in place which at least cover the basic functionality.